examples/sample-library-linking-project/roots/my-avail-root/App.avail/Foreign Interface.avail

```diff
@@ -18,7 +18,8 @@ Extends
             "string token lexer",
             "whole number lexer",
             "keyword lexer",
-            "operator lexer"
+            "operator lexer",
+            "After the current module is unloaded,⁇do_"
```

Comment:
You may also want to add `"After the current module is loaded,⁇do_"`.

Reply:
It was there. But I removed it and it worked. What does it do?

Reply:
When you call it with a block provided as the argument, the "unloaded" version ensures that the block is executed whenever the module is unloaded – after the added code and stuff from the module has been removed.   The newer "loaded" version postpones running a block of code until compilation has reached the end of the file, which is useful when defining new grammatical elements to export, but they would interfere with the existing grammar, e.g., creating a keyword lexer (for delimiting keywords like `"frog"` or `"foo_bar99"`) that's slightly different from the standard library's one.  Defining it during module compilation would lead to ambiguous parses because of the conflicting lexers.  By adding the new grammar right at the end (i.e., after all compilation has ended), and presumably exporting it in the Names section, subsequent modules can use that new grammar (and choose not to import the standard grammatical elements that interfere).